talk/morelanguage/move.tex

```diff
@@ -120,16 +122,16 @@
   \frametitlecpp[11]{Move semantics}
   \begin{block}{A few important points concerning move semantic}
     \begin{itemize}
-    \item the whole STL understands move semantics
-    \item move assignment operator is allowed to destroy source
+    \item the language and STL understand move semantics
+    \item move constructor and assignment operator are allowed to leave source "empty"
```

Comment:
I prefer the wording that the moved from object is left "empty" than destroyed, because the moved from object is in fact not destroyed, as no destructor is run.